results/google__Gemma-Embeddings-v0.8/d6813d20532a97ea8e30fc285397d5105316511f/ArguAna.json

```diff
@@ -0,0 +1,158 @@
+        "nauc_recall_at_1_std": -0.375886,
+        "nauc_recall_at_1_diff1": 0.433638,
+        "nauc_recall_at_3_max": 0.031895,
+        "nauc_recall_at_3_std": -0.425581,
+        "nauc_recall_at_3_diff1": 0.388713,
+        "nauc_recall_at_5_max": 0.191475,
+        "nauc_recall_at_5_std": -0.355495,
+        "nauc_recall_at_5_diff1": 0.39885,
+        "nauc_recall_at_10_max": 0.367349,
+        "nauc_recall_at_10_std": -0.443247,
+        "nauc_recall_at_10_diff1": 0.572029,
+        "nauc_recall_at_20_max": 0.408104,
+        "nauc_recall_at_20_std": -0.360184,
+        "nauc_recall_at_20_diff1": 0.569123,
+        "nauc_recall_at_100_max": 0.336931,
+        "nauc_recall_at_100_std": 0.82911,
+        "nauc_recall_at_100_diff1": 0.218345,
+        "nauc_recall_at_1000_max": 0.336931,
+        "nauc_recall_at_1000_std": 0.82911,
+        "nauc_recall_at_1000_diff1": 0.218345,
+        "nauc_precision_at_1_max": -0.139649,
+        "nauc_precision_at_1_std": -0.375886,
+        "nauc_precision_at_1_diff1": 0.433638,
+        "nauc_precision_at_3_max": 0.031895,
+        "nauc_precision_at_3_std": -0.425581,
+        "nauc_precision_at_3_diff1": 0.388713,
+        "nauc_precision_at_5_max": 0.191475,
+        "nauc_precision_at_5_std": -0.355495,
+        "nauc_precision_at_5_diff1": 0.39885,
+        "nauc_precision_at_10_max": 0.367349,
+        "nauc_precision_at_10_std": -0.443247,
+        "nauc_precision_at_10_diff1": 0.572029,
+        "nauc_precision_at_20_max": 0.408104,
+        "nauc_precision_at_20_std": -0.360184,
+        "nauc_precision_at_20_diff1": 0.569123,
+        "nauc_precision_at_100_max": 0.336931,
+        "nauc_precision_at_100_std": 0.82911,
+        "nauc_precision_at_100_diff1": 0.218345,
+        "nauc_precision_at_1000_max": 0.336931,
+        "nauc_precision_at_1000_std": 0.82911,
+        "nauc_precision_at_1000_diff1": 0.218345,
+        "nauc_mrr_at_1_max": -0.133793,
+        "nauc_mrr_at_1_std": -0.362766,
+        "nauc_mrr_at_1_diff1": 0.429598,
+        "nauc_mrr_at_3_max": -0.106856,
+        "nauc_mrr_at_3_std": -0.392887,
+        "nauc_mrr_at_3_diff1": 0.407134,
+        "nauc_mrr_at_5_max": -0.099574,
+        "nauc_mrr_at_5_std": -0.384355,
+        "nauc_mrr_at_5_diff1": 0.409777,
+        "nauc_mrr_at_10_max": -0.104527,
+        "nauc_mrr_at_10_std": -0.385681,
+        "nauc_mrr_at_10_diff1": 0.412832,
+        "nauc_mrr_at_20_max": -0.105995,
+        "nauc_mrr_at_20_std": -0.385301,
+        "nauc_mrr_at_20_diff1": 0.412397,
+        "nauc_mrr_at_100_max": -0.106958,
+        "nauc_mrr_at_100_std": -0.384759,
+        "nauc_mrr_at_100_diff1": 0.411898,
+        "nauc_mrr_at_1000_max": -0.106958,
+        "nauc_mrr_at_1000_std": -0.384759,
+        "nauc_mrr_at_1000_diff1": 0.411898,
+        "main_score": 0.83828,
+        "hf_subset": "default",
+        "languages": [
+          "eng-Latn"
+        ]
+      }
+    ]
+  },
+  "evaluation_time": -1,
```

Comment:
evaluation_time can't be -1. It should be null

Reply:
Thank you! I believe I have fixed this now.